engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -4126,15 +4126,24 @@ namespace dmGraphics
             VkCommandBuffer cmd_buffer = BeginSingleTimeCommands(context->m_LogicalDevice.m_Device, context->m_LogicalDevice.m_CommandPoolWorker);
 
             uint8_t tex_layer_count   = dmMath::Max(tex->m_LayerCount, ap.m_Params.m_LayerCount);
-            uint16_t tex_depth        = dmMath::Max(tex->m_Depth, ap.m_Params.m_Depth);
+            uint16_t tex_depth = dmMath::Max<uint16_t>( (uint16_t)1, dmMath::Max<uint16_t>(tex->m_Depth, ap.m_Params.m_Depth) );
```

Comment:
You can omit the type if all arguments are correct, i.e

dmMath::Max((uint16_t) 1, dmMath::Max(tex->m_Depth, ap.m_Params.m_Depth));